Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook

```diff
@@ -58,7 +55,8 @@
             "isHiddenWhenLocked": true,
             "timeContext": {
               "durationMs": 86400000
-            }
+            },
+            "value": "PE1"
```

Comment:
Is this hard code value? (Looks SID to me)
Please confirm its not hard coded value here

Reply:
SID is always passed by the parent workbook so the value set in the template internally is irrelevant.